.github/workflows/docker-build.yml

```diff
@@ -90,3 +90,44 @@ jobs:
           tags: ytorg/yotter:pypy
           cache-from: type=local,src=cache
           cache-to: type=local,dest=cache
+  nginx-build-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Get hash of latest image
+        run: docker pull nginx:1.19.6-alpine && docker inspect --format='{{index .RepoDigests 0}}' nginx:1.19.6-alpine > dockerhash.txt
+      - name: Write the current version to a file
+        run: "{ git describe --tags --abbrev=0 & date +\"%d-%m-%y\" & git rev-list HEAD --max-count=1 --abbrev-commit;} > version.txt"
+      - name: cache docker cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: ${{ github.workspace }}/cache
+          key: ${{ runner.os }}-docker-${{ hashFiles('**/dockerhash.txt') }}
```

Comment:
I think the cache key has to be changed to not conflict with other builds

Reply:
Actually no conflict already, but added suffix to make it more explicit